exercise 28.js

```diff
@@ -0,0 +1,17 @@
+/*
+. Implement the getCelciusConvertedToFahrenheit function
+
+it should accept one argument:
+
+a number
+
+the function should return the number converted from Celcius to Fahrenheit
+ */
+
+const farenheitDegree = 33.8
+function getCelciusConvertedToFahrenheit (number) {
+    return number * farenheitDegree;
```

Comment:
to chyba nie jest wlasciwy wzor - 10 stopni C to nie jest 338 stopni F, skad go masz?